src/main/java/com/eternalcode/parcellockers/configuration/implementation/PluginConfiguration.java

```diff
@@ -21,6 +21,25 @@ public static class Settings {
 
         @Description({" ", "# Whether the player after entering the server should receive information about the new version of the plugin?"})
         public boolean receiveUpdates = true;
+
+        @Description({" ", "# The URL to the database."})
+        public String databaseUrl = "localhost";
```

Comment:
To nie powinno się nazywać URL, URL jest wtedy kiedy dajesz tam wszystko żeby użytkownik sobie wypełnij tutaj jest to host